pkg/apis/policy/v1alpha1/replicascheduling_types.go

```diff
@@ -31,24 +31,6 @@ type ReplicaSchedulingSpec struct {
 	Preferences ClusterPreferences `json:"preferences"`
 }
 
-// ClusterPreferences describes weight for each cluster or for each group of cluster.
-type ClusterPreferences struct {
-	// StaticWeightList defines the static cluster weight.
-	// +required
-	StaticWeightList []StaticClusterWeight `json:"staticWeightList"`
-}
-
-// StaticClusterWeight defines the static cluster weight.
-type StaticClusterWeight struct {
-	// TargetCluster describes the filter to select clusters.
-	// +required
-	TargetCluster ClusterAffinity `json:"targetCluster"`
-
-	// Weight expressing the preference to the cluster(s) specified by 'TargetCluster'.
-	// +required
-	Weight int64 `json:"weight"`
-}
-
```

Comment:
Why move these types?

Reply:
I think that the replica scheduling_types.go may be deleted in the future ,but these two struct will still be in use in propagation_types.go，so I move it to propagation_types.go. If you think this is unnecessary, I will modify it.

Reply:
We haven't officially decided to deprecate the RSP API. So, please don't move it for now.

Reply:
ok